docs/asciidoc/static/offline-plugins.asciidoc

```diff
@@ -0,0 +1,53 @@
+[[offline-plugins]]
+=== Offline Plugin Management
+
+The Logstash <<working-with-plugins,plugin manager>> was introduced in the 1.5 release. This section discusses setting up
+local repositories of plugins for use on systems without access to the Internet.
+
+The procedures in this section require a machine running Logstash that has access to a public or private Rubygems server.
```

Comment:
> The procedures in this section require a machine running Logstash that has access to a public or private Rubygems server. 

Add - Lets call this a staging machine to bootstrap packages which is subsequently used for offline installation.